scalyr_agent/builtin_monitors/mysql_monitor.py

```diff
@@ -414,7 +416,8 @@ def _query(self, sql):
 
         try:
             self._cursor.execute(sql)
-        except pymysql.OperationalError, (errcode, msg):
+        except pymysql.OperationalError as xxx_todo_changeme:
+            (errcode, msg) = xxx_todo_changeme.args
```

Comment:
I think it is easier just to rename it properly rather than adding TODO because except naming it seems like normally ported code. I have made changes in all similar places.